community/modules/file-system/nfs-server/scripts/install-nfs-server.sh.tpl

```diff
@@ -12,14 +12,18 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set -ex
 
-yum -y install nfs-utils
 systemctl start nfs-server rpcbind
 systemctl enable nfs-server 
+
+# format and mount the disk. See https://cloud.google.com/compute/docs/disks/add-persistent-disk
+mkfs.ext4 -F -m 0 -E lazy_itable_init=0,lazy_journal_init=0,discard /dev/disk/by-id/google-attached_disk
 mkdir /exports
+mount -o discard,defaults /dev/disk/by-id/google-attached_disk /exports
```

Comment:
Since you are already changing this file, I would have made sure that this can run twice in a row (what happens on the next reboot? here you would loose all data, right?)

Also, I would prefer that the device is added to fstab.

This is easy to achieve in ansible, but there is code that checks for the presence in [fstab in here](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/file-system/pre-existing-network-storage/scripts/mount.sh)

Reply:
Added to fstab